lib/crowbar/client/command/upgrade/database.rb

```diff
@@ -50,7 +50,7 @@ def execute
                 err request.parsed_response["error"]
               end
 
-              response = JSON.parse(request.body)
+              response = JSON.parse(request.body) unless request.body.nil?
```

Comment:
yeah, i just realized that we have a pending submit request still hanging for that... :( https://build.suse.de/request/show/122514 It can't be merged right now it seems... :(